### PR TITLE
Fix packbits/unpackbits tests

### PR DESCRIPTION
**Pull request by skye**

### CODE REVIEW

File: tests/lax_numpy_test.py [hunk 1]
```diff
@@ -2252,7 +2256,7 @@ def onp_fun(*args):
            "_a_shape={}_axis={}_keepdims={}".format(
              jtu.format_shape_dtype_string(a_shape, a_dtype),
              axis, keepdims),
-         "a_rng": jtu.rand_default(), 
+         "a_rng": jtu.rand_default(),
```
Comment (hawkinsp) [inline tests/lax_numpy_test.py:2259]: It occurs to me that this is a bug too: we shouldn't instantiate rng factories in the test harness.

File: tests/lax_numpy_test.py [hunk 2]
```diff
@@ -2040,6 +2040,8 @@ def testRollaxis(self, shape, dtype, start, axis, rng_factory):
       for axis in [None, 0, 1, -2, -1]
       for rng_factory in [jtu.rand_some_zero]))
   def testPackbits(self, shape, dtype, axis, bitorder, rng_factory):
+    if numpy_version < (1, 17, 0):
```
Comment (hawkinsp) [inline tests/lax_numpy_test.py:2043]: Shouldn't we be testing the other arguments (i.e., the `bitorder=None` case?)

**Review by hawkinsp — CHANGES_REQUESTED** [review]

**Comment by skye** [comment]
I'll address these comments in a follow-up PR, trying to fix these tests asap.

File: tests/lax_numpy_test.py (hunk 2 above)
Comment (skye) [reply tests/lax_numpy_test.py:2043]: Turns out `bitorder` should only be "big" or "little" and defaults to "big".

**Comment by jakevdp** [comment]
Would it be worth adding travis test coverage for older numpy versions to catch issues like this? If we use num_generated_cases=1, it would not consume too many resources.

**Comment by skye** [comment]
Perhaps! I already feel very impatient waiting for the travis presubmit to run, so I wouldn't wanna add more than a few seconds to it, but I bet if we only run lax_numpy_test.py with num_generated_cases=1 we'd get good bang for our buck.

**Pull request merged**